spec/index.bs

```diff
@@ -7043,6 +7055,17 @@ enum GPUStoreOp {
                 If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
                 <div class=validusage>
                     - It is [$valid to draw$] with |this|.
+                    - Let |buffers| be |this|.{{GPURenderEncoderBase/[[pipeline]]}}.{{GPURenderPipeline/[[descriptor]]}}.{{GPURenderPipelineDescriptor/vertex}}.{{GPUVertexState/buffers}}.
+                    - For each {{GPUIndex32}} |slot| `0` to |buffers|.length:
```

Comment:
This goes back to the discussion we had on Monday. We can detect this class of API misuse, so we should.
Just because there is this RBA hammer doesn't mean that it's OK to apply averywhere.

Reply:
We can always relax later too.